services/server/schemas.py

```diff
@@ -10,3 +10,8 @@ class User(UserBase):
 
     class Config:
         orm_mode = True
+
```

Comment:
Please pull from this branch [https://github.com/NBNARADHYA/prep-project-4.1.2/tree/relations](https://github.com/NBNARADHYA/prep-project-4.1.2/tree/relations) and update your pr, and change your code accordingly

Reply:
This what I have done already